Assets/AnkrSDK/Plugins/WalletConnectSharp.Unity/WalletConnect.WalletConnectTestWrapper.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using AnkrSDK.WalletConnectSharp.Core;
+using AnkrSDK.WalletConnectSharp.Core.Models;
+using Cysharp.Threading.Tasks;
+
+namespace AnkrSDK.WalletConnectSharp.Unity
+{
+	public partial class WalletConnect
```

Comment:
Why do you make wallet connect partial? Doesn't introduce any useful behaviour

Reply:
Because I use a nested class WalletConnectTestWrapper in order to wrap WalletConnect to use its private methods in WalletConnectTests. This allows to test certain scenarios and keep methods used in those scenarios private for a user of WalletConnect

Reply:
You are still making walletconnecttestwrapper public. Solve this by adding internal methods for test and add internalsvisibleattribute to an assembly https://learn.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.internalsvisibletoattribute?view=net-7.0